common/flavorization/wc_title_holders.txt

```diff
@@ -4,6 +4,26 @@
 @county_cultural_prio = 18
 @barony_cultural_prio = 7
 
+blood_troll_patriarch_male = {
```

Comment:
You should define a tier. Otherwise, every vassal will be called Ma'da.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/126676109-dced2bde-e73d-4c76-85f2-2c8a4cff5169.png)

</details>

Reply:
Added in 1b9ca25949cda89b2be8502e5f286054cdf88f70.